### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by 2103364** [Chia-Network/chia-blockchain#3351] [Support]
Created a wallet on computer # 1. Conducted a mnemonic and opened a wallet on computer # 2. I reinstalled the system on computer # 1 and when entering mnemonics I get the message "chia invalid order of mnemonic". I checked everything many times, when typing words they appear in the drop-down list, but the wallet did not connect 

WIN10 LTSC
Xeon 2030L

**Comment by 2103364** [comment]
wallet on computer # 2 starts, can I transfer the wallet by copying?

**Comment by 2103364** [comment]
I will clarify. I have the latest version of the program

**Comment by redskinhu** [comment]
Hello

I have similar issue.

- Key generated on the 1st PC - OK
- Imported to 2nd PC - OK
- Imported to 3nd PC - OK
- Tried to import to the 4th PC, error message: "invalid order of mnemonic words" (tried 5 times)

1st PC: Win10 19042.928, Chia 1.1.4
2nd PC: Debian 10, Chia 1.1.5.dev34
3rd PC: Win10 19041.804, Chia 1.1.4
4th PC: Win10 19042.928, Chia 1.1.4

**Comment by 2103364** [comment]
in windows firewall give permission to all chia programs

вс, 9 мая 2021 г., 21:17 Tony ***@***.***>:

**Reply by redskinhu** [reply]
Problem solved: I need glasses...
`frame != fame`

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
